doc/api/crypto.md

```diff
@@ -2141,6 +2139,95 @@ threadpool request. To minimize threadpool task length variation, partition
 large `randomFill` requests when doing so as part of fulfilling a client
 request.
 
+### crypto.scrypt(password, salt, keylen[, options], callback)
+<!-- YAML
+added: REPLACEME
+-->
+- `password` {string|Buffer|TypedArray}
+- `salt` {string|Buffer|TypedArray}
+- `keylen` {number}
+- `options` {Object}
+  - `N` {number} CPU/memory cost parameter. Must be a power of two greater
+                 than one. **Default:** `16384`
+  - `r` {number} Block size parameter. **Default:** `8`
+  - `p` {number} Parallelization parameter. **Default:** `1`
+  - `maxmem` {number} Memory upper bound. It is an error when (approximately)
+                      `128*N*p*r > maxmem` **Default:** `32 * 1024 * 1024`
```

Comment:
Note to self: typo, should be `128*N*r`.  Happens in another place in this file and in the test.